dumps/obj_fetch_non-key.c

```diff
@@ -0,0 +1,218 @@
+/* fetch object of non-existing dkey after successfull sequence of update - fetch w/o transaction */
```

Comment:
Can you please start the file with a command line allowing us to reproduce the sequence?